src/App.js

```diff
@@ -25,18 +27,23 @@ function App() {
 
   return (
     <Router>
-      <Fragment>
+      <Fragment>   
         <nav className="navbar navbar-light bg-light static-top">
           <div className="container">
             <Link className="navbar-brand" to="/">
               The Name (Heritage) Project.
             </Link>
             <a className="btn btn-primary" href="#">
-              Become Volunteer
+              Become Volunteer     
+            </a>
+            <Link to="/volunteerList">
+            <a className="btn btn-primary" href="#">
+              See Volunteers     
             </a>
+            </Link>
           </div>
         </nav>
-
+        <Route path="/volunteerList" component={volunteerList} />
```

Comment:
to be consistent, can you make the path volunteer-list than volunteerList please?